src/wallet/wallet2.cpp

```diff
@@ -5035,7 +5035,7 @@ std::string wallet2::make_uri(const std::string &address, const std::string &pay
   }
 
   std::string uri = "monero:" + address;
-  bool n_fields = 0;
+  uint8_t n_fields = 0;
```

Comment:
I hate to be so picky, but 256 parameters is plausibly reached, as this will be automated. Please use, eg, unsigned int.